.github/workflows/main.yml

```diff
@@ -14,28 +14,39 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - python-version: 3.6
+          - python-version: "3.7"
             env:
               TOXENV: py
-          - python-version: 3.7
+          - python-version: "3.8"
             env:
               TOXENV: py
-          - python-version: 3.8
+          - python-version: "3.9"
             env:
               TOXENV: py
-          - python-version: 3.9
+          - python-version: "3.10"
             env:
               TOXENV: py
+          - python-version: "3.11"
+            env:
+              TOXENV: py
+          - python-version: "3.11"
+            env:
+              TOXENV: flake8
+          - python-version: "3.11"
+            env:
+              TOXENV: latest
```

Comment:
We again have both `latest` and `py` for 3.11, which do the same.

You can remove the `latest` entry here, and change the `if` to expect `py` instead of `latest`.